src/webui/www/public/login.html

```diff
@@ -6,60 +6,58 @@
-                new Event(e).stop();
-                submitLoginForm();
+            document.getElementById('loginform').addEventListener('submit', function(e) {
+                e.preventDefault();
             });
         });
 
         function submitLoginForm() {
-            new Request({
-                url: 'api/v2/auth/login',
-                method: 'post',
-                data: $('loginform').toQueryString(),
-                onComplete: function() {
-                    $('password').set('value', '');
-                },
-                onFailure: function(xhr) {
-                    if (xhr.responseText !== "") {
-                        $('error_msg').set('html', xhr.responseText);
-                    }
-                    else {
-                        $('error_msg').set('html', 'QBT_TR(Unable to log in, qBittorrent is probably unreachable.)QBT_TR[CONTEXT=HttpServer]');
-                    }
-                },
-                onSuccess: function(text) {
-                    if (text == "Ok.") {
-                        // Session started. Simply reload page.
-                        window.location.reload();
-                    }
-                    else {
-                        $('error_msg').set('html', 'QBT_TR(Invalid Username or Password.)QBT_TR[CONTEXT=HttpServer]');
-                    }
+            var errorMsgElement = document.getElementById('error_msg');
+
+            var xhr = new XMLHttpRequest();
+            xhr.open('POST', 'api/v2/auth/login', true);
+            xhr.setRequestHeader('Content-type', 'application/x-www-form-urlencoded; charset=utf-8');
+            xhr.addEventListener('readystatechange', function() {
+                if (xhr.readyState === 4) {  // DONE state
+                    if ((xhr.status === 200) && (xhr.responseText === "Ok."))
+                        location.reload(true);
+                    else
+                        errorMsgElement.innerHTML = 'QBT_TR(Invalid Username or Password.)QBT_TR[CONTEXT=HttpServer]';
                 }
-            }).send();
+            });
+            xhr.addEventListener('error', function() {
+                errorMsgElement.innerHTML = (xhr.responseText !== "")
+                    ? xhr.responseText
+                    : 'QBT_TR(Unable to log in, qBittorrent is probably unreachable.)QBT_TR[CONTEXT=HttpServer]';
+            });
+
+            var usernameElement = document.getElementById('username');
+            var passwordElement = document.getElementById('password');
+            var queryString = "username=" + encodeURIComponent(usernameElement.value) + "&password=" + encodeURIComponent(passwordElement.value);
+            xhr.send(queryString);
+
+            // clear the field
+            passwordElement.value = '';
         }
     </script>
 </head>
 
 <body>
+    <noscript id="noscript"><h1>QBT_TR(JavaScript Requried! You must enable JavaScript for Web UI to work properly)QBT_TR[CONTEXT=HttpServer]</h1></noscript>
```

Comment:
Can you add `the` between `for Web UI`? Thus, `You must enable JavaScript for the Web UI to work properly`